src/components/movie-card/movie-card.jsx

```diff
@@ -1,8 +1,32 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 export class MovieCard extends React.Component {
   render() {
     const { movie, onMovieClick } = this.props;
-    return <div className="movie-card" onClick={() => { onMovieClick(movie); }}>{movie.Title}</div>;
+
+    return (
+    <div onClick={() => { onMovieClick(movie) }} className='movie-card'>{movie.Title}</div>
+    );
   }
-}
\ No newline at end of file
+}
+
+MovieCard.propTypes = {
+  movie: PropTypes.shape({
+    Title: PropTypes.string.isRequired,
+    Description: PropTypes.string.isRequired,
+    ImagePath: PropTypes.string.isRequired
+    // Genre: PropTypes.shape({
+    //   Name: PropTypes.string.isRequired,
+    //   Description: PropTypes.string.isRequired
+    // }),
+    // Director: PropTypes.shape({
+    //   Name: PropTypes.string.isRequired,
+    //   Bio: PropTypes.string.isRequired,
+    //   BirthYear: PropTypes.string.isRequired,
+    //   DeathYear: PropTypes.string.isRequired
+    // }),
+    // Featured: PropTypes.bool.isRequired
+  }).isRequired,
+  onMovieClick: PropTypes.func.isRequired
+};
```

Comment:
This must be the source of the prototype error you are getting. This component expects the movie prop object to contain a Title but I think the one passed doesn't. Try to `console.log(`movie: `, movie);`